include/fs/fs.h

```diff
@@ -173,6 +180,8 @@ struct fs_statvfs {
  *
  * @retval 0 on success;
  * @retval -EINVAL when a bad file name is given;
+ * @retval -EROFS if file is read-only, or when file system has been mouted with
```

Comment:
this is all about when file was opened with the "write" flag

Reply:
s/mouted/mounted/g;

Reply:
> this is all about when file was opened with the "write" flag

Unfortunately I do not understand.

Reply:
clarified off-channel

Reply:
I didn't understand either, but from the patch now I do.  +1.